docm_filter.pl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use feature qw(say);
+
+die("wrong number of arguments") unless scalar(@ARGV) == 2;
+my ($docm_out_vcf, $outdir) = @ARGV;
+
+open(my $docm_vcf_fh, $docm_out_vcf)
+    or die("couldn't open $docm_out_vcf to read");
+open(my $docm_filter_fh, ">", "$outdir/docm_filter_out.vcf")
+    or die("couldn't open docm_filter_out.vcf for write");
+
+while (<$docm_vcf_fh>) {
+    chomp;
+    if (/^#/) {
+        say $docm_filter_fh $_;
+    }
+    else {
+        my @columns = split /\t/, $_;
+        my @tumor_info = split /:/, $columns[10];
+        my ($AD, $DP) = ($tumor_info[1], $tumor_info[2]);
```

Comment:
So far it seems like Haplotypecaller consistently makes format GT:AD:DP:GQ:PL in output vcf. This is just a way to work around filter issue and be very specific with docm variant caller. If docm variant caller is changed, docm_filter.pl will be modified too. Since docm variant caller is set for long time, the specific docm_filter.pl seems ok.